geode-core/src/integrationTest/java/org/apache/geode/internal/net/SSLSocketHostNameVerificationIntegrationTest.java

```diff
@@ -211,18 +211,17 @@ private Thread startServerNIO(final ServerSocket serverSocket, int timeoutMillis
       } catch (Throwable throwable) {
         serverException = throwable;
       } finally {
-        if (engine != null && socket != null) {
+        if (engine != null) {
```

Comment:
null check is no longer needed?

Reply:
Intellij informed me that if (engine != null) it isn't possible that (socket == null).  I agreed & removed that check.

Reply:
what a helpful tool!